server/index.js

```diff
@@ -13,18 +13,62 @@ app.use((req, res, next) => {
   next();
 });
 
-
+//working
 app.get('/api/description/:productId', (req, res) => {
   const productId = req.params.productId;
-  console.log(productId);
   models.getOneItem(productId)
-    .then((value) => res.status(200).json(value))
+    .then((val) => res.status(200).json(val))
+    .catch((err) => {
+      res.status(400)
+      res.send(err);
```

Comment:
Just for consistency, you may want to combine these onto one line ( res.status(400).send(err); ) like you have in your other routes.